lib/kubernetes/eirini_client.rb

```diff
@@ -0,0 +1,27 @@
+require 'steno'
+require 'kubernetes/kube_client_builder'
+require 'cloud_controller/errors/api_error'
+
+module Kubernetes
```

Comment:
Would be preferable if we put these methods instead in `Kubernetes::ApiClient` for consistency (all of our other Kubernetes interactions currently go through that class)

Would be happy to discuss refactoring that after though if we'd like.